samples/snippet/job_search_autocomplete_job_title.js

```diff
@@ -0,0 +1,72 @@
+// DO NOT EDIT! This is a generated sample ("Request",  "job_search_autocomplete_job_title")
+'use strict';
+
+// [START job_search_autocomplete_job_title]
+// [START job_search_autocomplete_job_title_core]
+
+const talent = require('@google-cloud/talent').v4;
+
+/**
+ * Complete job title given partial text (autocomplete)
+ *
+ * @param projectId {string} Your Google Cloud Project ID
+ * @param tenantId {string} Identifier of the Tenantd
+ */
+function sampleCompleteQuery(projectId, tenantId, query, numResults, languageCode) {
+  const client = new talent.CompletionClient();
+  // const projectId = 'Your Google Cloud Project ID';
+  // const tenantId = 'Your Tenant ID (using tenancy is optional)';
+  // const query = '[partially typed job title]';
+  // const numResults = 5;
+  // const languageCode = 'en-US';
+  const formattedParent = client.tenantPath(projectId, tenantId);
+  const languageCodes = [languageCode];
+  const request = {
+    parent: formattedParent,
+    query: query,
+    pageSize: numResults,
+    languageCodes: languageCodes,
+  };
+  client.completeQuery(request)
+    .then(responses => {
+      const response = responses[0];
+      for (const result of response.completionResults) {
+        console.log(`Suggested title: ${result.suggestion}`);
+        // Suggestion type is JOB_TITLE or COMPANY_TITLE
+        console.log(`Suggestion type: ${result.type}`);
+      }
+    })
+    .catch(err => {
+      console.error(err);
+    });
+}
+
+
+// [END job_search_autocomplete_job_title_core]
+// [END job_search_autocomplete_job_title]
+// tslint:disable-next-line:no-any
+
+const argv = require(`yargs`)
```

Comment:
We've moved away from using `yargs` in our samples, as it adds an additional dependency users need to worry about.

It doesn't look like anything too fancy is being done with yargs, so I think we could probably just drop the dependency, opting for `process.argv.slice(2)`?